db/migrate/20190507103007_add_education_attr_to_workflows.rb

```diff
@@ -0,0 +1,5 @@
+class AddEducationAttrToWorkflows < ActiveRecord::Migration
+  def change
+    add_column :workflows, :education_api, :boolean, default: false
```

Comment:
I'm not super happy with the attribute name, ideas welcome. `education_use`, `education_only`?

Reply:
I'd go `education_only` since it amounts to a filter. Still pretty service-specific, which we don't have a lot of elsewhere, though. `hidden` or `omit` ?

Reply:
Could you maybe make it something more generic so it's not tied specifically to the education API? Just thinking there might be other reasons to use this in the future and having "education" in the name doesn't make it very clear what the attribute actually does.

Maybe call it `serialize` and reverse the logic, so it defaults to true?

Reply:
Thanks @adammcmaster - i like the invert paradigm and naming suggestion, I'll update.